core/src/main/resources/hudson/slaves/SimpleScheduledRetentionStrategy/config.jelly

```diff
@@ -36,6 +36,6 @@ THE SOFTWARE.
                checkMessage="${%Scheduled Uptime is mandatory and must be a number.}"/>
     </f:entry>
     <f:entry>
-        <f:checkbox title="${%Keep online while jobs are running}" help="/help/system-config/master-slave/demand/keepUpWhenActive.html" name="retentionStrategy.keepUpWhenActive" checked="${h.defaultToTrue(instance.keepUpWhenActive)}" />
+        <f:checkbox title="${%Keep online while builds are running}" help="/help/system-config/master-slave/demand/keepUpWhenActive.html" name="retentionStrategy.keepUpWhenActive" checked="${h.defaultToTrue(instance.keepUpWhenActive)}" />
```

Comment:
Not sure we want to derail localizations just by that. Better to add an English-only override (will look like a hack, but will do the job)